pkg/api/api.go

```diff
@@ -618,3 +618,11 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/api/snapshots-delete/:deleteKey", reqSnapshotPublicModeOrSignedIn, routing.Wrap(hs.DeleteDashboardSnapshotByDeleteKey))
 	r.Delete("/api/snapshots/:key", reqSignedIn, routing.Wrap(hs.DeleteDashboardSnapshot))
 }
+
+func (hs *HTTPServer) shouldUseK8sAPI(resource string) bool {
+	unifiedStorageOptions := hs.Cfg.UnifiedStorage
+	if _, ok := unifiedStorageOptions[resource]; ok {
```

Comment:
If we want to disable the usage of K8sAPI for a given resource, the only way would be to remove the entire section...

would it make sense to parse a field called `enabled` (default true)?

Reply:
Let's add this on a follow-up PR
c/c @leonorfmartins